lib/wicked_pdf/railtie.rb

```diff
@@ -5,20 +5,16 @@
 class WickedPdf
   if defined?(Rails.env)
 
-    if Rails::VERSION::MAJOR >= 5
+    if Rails::VERSION::MAJOR >= 4
```

Comment:
The only difference between the Rails 4 and 5+ paths was `:include` vs `prepend`, so they were consolidated, and feature detection is now being used to determine whether to use `:prepend` or `:include`.